python_modules/dagster/dagster_tests/asset_defs_tests/test_asset_defs_source_metadata.py

```diff
@@ -0,0 +1,72 @@
+import os
+from typing import cast
+
+from dagster import AssetsDefinition, load_assets_from_modules
+from dagster._core.definitions.metadata import (
+    DEFAULT_SOURCE_FILE_KEY,
+    LocalFileSource,
+    with_code_source,
+)
+from dagster._utils import file_relative_path
+
+
+def test_asset_code_origins() -> None:
+    from dagster_tests.asset_defs_tests import asset_package
+
+    from .asset_package import module_with_assets
+
+    collection = load_assets_from_modules([asset_package, module_with_assets])
+
+    for asset in collection:
+        if isinstance(asset, AssetsDefinition):
+            for key in asset.keys:
+                assert "dagster/source_paths" not in asset.metadata_by_key[key]
```

Comment:
I don't see this string elsewhere in the PR., Is that expected?

Reply:
It's just the concatenation of the namespace `dagster` with the `DagsterModel` property name `source_paths` which happens in the metadata set body - it's not explicitly defined in the same way e.g. `dagster/column_lineage` appears only in tests